README.md

```diff
@@ -26,34 +34,41 @@ $ sudo apt-get install -y nodejs
 1. Set up Git on Windows: https://git-scm.com/downloads
 2. Install through the website: https://nodejs.org/en/download/
 
-## Setup
+---
+
+## 2. Setup
 
 Open a terminal window (`Terminal` or `iTerm2` on Mac, `Git Bash` or `Ubuntu on Windows`) at the root of the project, and type:
 
 ```
-$ npm install
+$ yarn
```

Comment:
Don't you need to do 'brew install yarn' first?

Reply:
Good point, lemme add that to the instructions!